dataline-webapp/src/locales/en.json

```diff
@@ -38,8 +38,20 @@
   "form.user": "User",
   "form.password": "Password",
   "form.dbname": "DB Name",
+  "form.postgres_host": "Host",
```

Comment:
discussed during the sync. ideally `*/specifications/get` endpoint should display exactly the data that is needed with the correct display names. we will leave things like they are for right now, but then figure this out as we go into testing.